index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jim Ntare | Software Engineer</title>
+    <link rel="stylesheet" href="./build/styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="nav_bar flex">
+        <div class="logo">My Logo</div>
```

Comment:
- [ ] [OPTIONAL] I highly recommend you make this change. The logo should be clickable. kindly wrap it in an `a`.